runbotics/runbotics-orchestrator-ui/src/src-app/theme/cronStyles.css

```diff
@@ -14,6 +14,7 @@
     margin-bottom: 10px;
 }
 .react-js-cron-field > span {
+    /* styled component in Cron.styles.ts */
```

Comment:
is it still used via css rather than styled-component? if not then remove it

Reply:
removed 😅